app/javascript/bali/controllers/datepicker-controller.js

```diff
@@ -137,4 +139,8 @@ export class DatepickerController extends Controller {
 
     this.flatpickr.setDate(currentDate)
   }
+
+  isWeekend = date => {
+    return date.getDay() === 0 || date.getDay() === 6
```

Comment:
Does the `sunday` and `saturday` always return `0`and `6`? ... Just wondering if this is not shadowing an error again...


https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getUTCDay

Reply:
In flatpickr documentation it uses the number 1 to indicate Monday, and 0 to indicate Sunday
I assume this will never change since it does not indicate otherwise in their example:
https://flatpickr.js.org/examples/#disabling-dates-by-a-function